students/mroz_sebastian/lesson_01_python_basics/l2_snakify/Fractional_part.py

```diff
@@ -0,0 +1,4 @@
+from math import floor 
+
+a = float( input() )
+print( a - floor( a ) )
```

Comment:
It's better to avoid doing math operations in print().
Consider using the following solution:

number = float(input("Enter float number: "))
fractional = number - int(number)
print(fractional)